dotnetv3/Cognito/Actions/HelloCognito.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[Cognito.dotnetv3.HelloCognito]
+
+namespace CognitoActions;
+
+/// <summary>
+/// A class that introduces the Cognito Identity Provider by listing the
+/// user pools for the account.
+/// </summary>
+public class HelloCognito
+{
+    private static ILogger logger = null!;
+
+    static async Task Main(string[] args)
+    {
+        // Set up dependency injection for Amazon Cognito.
+        using var host = Host.CreateDefaultBuilder(args)
+            .ConfigureLogging(logging =>
+                logging.AddFilter("System", LogLevel.Debug)
+                    .AddFilter<DebugLoggerProvider>("Microsoft", LogLevel.Information)
+                    .AddFilter<ConsoleLoggerProvider>("Microsoft", LogLevel.Trace))
+            .ConfigureServices((_, services) =>
+                services.AddAWSService<IAmazonCognitoIdentityProvider>()
+                .AddTransient<CognitoWrapper>()
+            )
+            .Build();
+
+        logger = LoggerFactory.Create(builder => { builder.AddConsole(); })
+            .CreateLogger<HelloCognito>();
+
+        var cognitoWrapper = host.Services.GetRequiredService<CognitoWrapper>();
```

Comment:
For the hello service example, in my opinion we should directly call the service client. The idea is to provide the simplest use example, and so I wouldn't use the wrapper here. You can see Aurora, Route53, or CloudWatch for some examples.

Reply:
Moved code to the HelloGlue class using a paginator.